analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/HashesShouldHaveUnpredictableSalt.CSharp9.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Security.Cryptography;
+﻿using System;
```

Comment:
```suggestion

```

Reply:
It's needed for the type `System.Action` in the [StaticLambda method](https://github.com/SonarSource/sonar-dotnet/blob/ed9df5fedaf9a2f04f9361f486d67b187a202d7f/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/HashesShouldHaveUnpredictableSalt.CSharp9.cs#L39).